### PR TITLE
Update for creating real testbed

### PR DESCRIPTION
**Pull request by hmunfru**
#### REVIEWER
@jesuspg 
#### DESCRIPTION
Change for updating domain default in the testbed. Update all the environment variables.
### Test
Locally for parts. It is needed to be merged to develop to be able to be testbed from cloud-init

### CODE REVIEW

File: tests/install_testbed/register_region.py [hunk 1]
```diff
@@ -151,6 +151,15 @@ def region_exists(self, region_id):
         if not self.is_region(region_id):
             self.keystone.regions.create(region_id)
 
+    def change_domain_name(self):
+        """ Ensure that the region exists: create if it does not.
```
Comment (flopezag) [inline tests/install_testbed/register_region.py]: Something is wrong in the description of this method

File: tests/install_testbed/register_region.py [hunk 2]
```diff
@@ -151,6 +151,15 @@ def region_exists(self, region_id):
...
+
+        :param region_id: the region id (the region name)
+        :return: Nothing
+        """
+        domain = self.keystone.domains.find(name="Default")
```
Comment (flopezag) [inline tests/install_testbed/register_region.py]: You was changing are Default by default previously, so why do you need it?

**Comment by flopezag** [comment]
You are launching pep8 in the Travis execution, apart that it produced and error, you should use pdi-pep8 and not pep8

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**